nilearn/plotting/displays.py

```diff
@@ -584,7 +584,7 @@ def add_overlay(self, img, threshold=1e-6, colorbar=False, **kwargs):
         kwargs.setdefault('interpolation', 'nearest')
         ims = self._map_show(img, type='imshow', threshold=threshold, **kwargs)
 
-        if colorbar:
+        if colorbar and ims:
```

Comment:
Can you also address this comment please :)

I am thinking about people looking at the code in the future.